src/Reader/Report/Impl/XmlCsob/Entity/Report.php

```diff
@@ -13,6 +13,10 @@
 final class Report implements IReport
 {
 
+	// positive / negative balance indicator
+	private const INDICATOR_C = 'C';
+	private const INDICATOR_D = 'D';
```

Comment:
V interfacu by to bylo lepsi + "INDICATOR_*" je dost nic nerikajici. CREDIT/DEBIT by byl asi lepsi. A pozor na to aby zde nebyly jeste STORNO stavy jako u Aviz, tedy "RC" a "RD".

Reply:
Storno zde byt nemůže, jedná se o označení kladného nebo mínusového zůstatku na účtu (přichází v úvahu pouze má účet kontokorent)